src/views/SelectProduct.vue

```diff
@@ -222,20 +240,90 @@ export default defineComponent({
       products: 'product/getProducts',
       isScrollable: 'product/isScrollable',
       appliedFilters: 'product/getAppliedFilters',
-      query: 'product/getQuery'
+      query: 'product/getQuery',
+      pendingJobs: 'job/getPendingJobs',
+      getTagsAndOperator: 'job/getTagsAndOperator',
+      getCurrentEComStore:'user/getCurrentEComStore',
+      userProfile: 'user/getUserProfile',
+      currentEComStore: 'user/getCurrentEComStore',
+      shopifyConfig: 'util/getShopifyConfig',
+      facilitiesByProductStore: 'util/getFacilityByProductStore',
     })
   },
   data () {
     return {
       threshold: '' as any,
-      queryString: ''
+      queryString: '',
+      isFilterChanged: false,
+      isServiceScheduling: false,
+      job: {} as any,
+      jobId: "" as any
+    }
+  },
+  async ionViewWillEnter(){
+    if (this.$route.query.id) {
+      this.jobId = this.$route.query.id;
+      let job = this.pendingJobs.find((job: any) => job.jobId === this.jobId)
+      job = job ? job : await JobService.fetchJob({eComStoreId: this.getCurrentEComStore.productStoreId, jobId: this.jobId})
+      if (job) {
+        this.job = job;
+        if (job.runtimeData?.searchPreferenceId) {
+          const includedTags = this.getTagsAndOperator(job.runtimeData.searchPreferenceId, "included").tags
```

Comment:
Store `this.getTagsAndOperator(job.runtimeData.searchPreferenceId, "included")` in a variable and then use